backend/whisper_transcriber.py

```diff
@@ -0,0 +1,100 @@
+import os
+import sys
+import numpy as np
+import whisper_timestamped as whisper
+from pyannote.core import Segment
+from contextlib import contextmanager
+import logging
+from config import NON_SPECIFIC_MODELS
+
+
+# FULL CREDIT TO JUANMA CORIA FOR THIS DIART IMPLEMENTATION WITH WHISPER (COLOR YOUR CAPTIONS ON MEDIUM)
+
+
+@contextmanager
+def suppress_stdout():
+    # Auxiliary function to suppress Whisper logs (it is quite verbose)
+    # All credit goes to: https://thesmithfam.org/blog/2012/10/25/temporarily-suppress-console-output-in-python/
```

Comment:
Redundant

Reply:
If you're referring to the function itself, not using it would lead to console pollution. Here's an example of the output it prevents when transcribing an audio segment:
```
[00:00.000 --> 00:00.260] Okay.
[00:00.420 --> 00:00.920] Just
[00:00.920 --> 00:01.020] let
[00:01.020 --> 00:01.100] me
[00:01.100 --> 00:01.200] know
[00:01.200 --> 00:01.380] what's
[00:01.380 --> 00:01.480] up.
```

Reply:
I referred to the comment about the credit. We (developers) always copy from one another. No need to provide credit, It's not the Oscar here :)